src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -26,7 +28,9 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
         const mostWeightedDistrict = data.find((entry) => entry.votesPerSeat === highestVotingPower)!.name;
         const leastWeightedDistrict = data.find((entry) => entry.votesPerSeat === lowestVotingPower)!.name;
         const calculateVulnerable = isQuotientAlgorithm(this.props.algorithm);
-        const mostVulnerable = calculateVulnerable ? getMostVulnerableSeatByQuotient(data) : undefined;
+        const mostVulnerable = calculateVulnerable
+            ? getMostVulnerableSeatByQuotient(data, this.props.nationalPartyResults, this.props.districtThreshold)
+            : undefined;
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5991602)